### PR TITLE
report: remove sparkline

### PR DESCRIPTION
**Pull request by Hoten**
#8185

### CODE REVIEW

File: lighthouse-core/report/html/renderer/performance-category-renderer.js [hunk 1]
```diff
@@ -72,22 +71,18 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
...
-      displayEl.title = displayValue;
+      displayEl.title = audit.result.displayValue;
     }
 
     return element;
   }
 
   /**
-   * Get an audit's wastedMs to sort the opportunity by, and scale the sparkline width
+   * Get an audit's wastedMs to sort the opportunity by.
```
Comment (exterkamp) [inline lighthouse-core/report/html/renderer/performance-category-renderer.js]: This is worded clunky, I tried to make it a little better.
```suggestion
   * Get an audit's wastedMs, for sorting by in an opportunity table.
```

**Comment by Hoten** [comment]
@googlebot no one likes you

**Comment by paulirish** [comment]
Gotta say, I think the report is weaker without them. :/

**Comment by patrickhulce** [comment]
😢 I'm gonna find it hard to approve this one

**Comment by patrickhulce** [comment]
Are we keeping them? :D

**Comment by Hoten** [comment]
yeah, but hide in mobile

**Pull request closed without merging**